packages/common/src/models/ErrorReporting.ts

```diff
@@ -1,13 +1,45 @@
 export enum ErrorLevel {
-  'Warning' = 'Warning',
+  /**
+   * FATAL
+   * What to use for: anything that results in app crash / "Something went wrong"
+   * These are high prio for on-call to look into, and generally should trigger an alert of some kind.
+   * Applies to scenarios that are completely unrecoverable other than to refresh client
+   */
   'Fatal' = 'Fatal',
-  'Debug' = 'Debug',
+
+  /**
+   * ERROR
+   * What to use for: Unexpected breakage but recoverable for the user.
+   * e.g. click button but fails & shows error toast. Generally applies things that can be "retried" by the user
+   */
   'Error' = 'Error',
+
+  /**
+   * WARNING
+   * What to use for: Something not ideal but within realm of expected behavior
```

Comment:
a minor or expected failure that may be concerning if spiking or occurring too frequently